vhdbuilder/packer/windows-image.env

```diff
@@ -4,6 +4,6 @@
 # CLI example to get the latest image version:
 #   az vm image show --urn MicrosoftWindowsServer:WindowsServer:2019-Datacenter-Core-with-Containers-smalldisk:latest
 WINDOWS_2019_BASE_IMAGE_SKU=2019-Datacenter-Core-smalldisk
-WINDOWS_2019_BASE_IMAGE_VERSION=17763.1757.2102060435
+WINDOWS_2019_BASE_IMAGE_VERSION=17763.1817.2103030313
```

Comment:
Updated the image version while keeping the 3B patch in the test.

Reply:
17763.1817 is 3B. I think that we can use the build number to identify the patch.
https://github.com/Azure/AgentBaker/blob/master/vhdbuilder/release-notes/AKSWindows/2019/17763.1817.210310.txt

Reply:
021-03-29T07:14:51.7975596Z [0;32m    azure-arm: System Info[0m
2021-03-29T07:14:51.7984822Z [0;32m    azure-arm: 	OS Name        : Windows Server 2019 Datacenter[0m
2021-03-29T07:14:51.7989363Z [0;32m    azure-arm: 	OS Version     : 17763.1817[0m
2021-03-29T07:14:51.7999723Z [0;32m    azure-arm: 	OS InstallType : Server Core[0m

Seems to be good, from the log of a build, https://msazure.visualstudio.com/CloudNativeCompute/_build/results?buildId=40866649&view=logs&j=1a85a368-fe74-521a-64cd-10ad659b69f5&t=38d2a6c7-b585-5636-a3e3-9b23ffb30747